docs/custom-image.md

```diff
@@ -0,0 +1,115 @@
+In this section we will learn how we can create a custom BitOps image with plugins and additional utilities. 
+
+
+
+### Clone Bitops source
+First thing is first, if you haven't already clone the github bitops repo locally
+`git clone https://github.com/bitovi/bitops.git`
```

Comment:
this isn't necessary.  it should say something like "create a repo for your custom bitops image"